src/test/scala/mesosphere/marathon/api/akkahttp/v2/EventsControllerTest.scala

```diff
@@ -125,8 +125,11 @@ class EventsControllerTest extends AkkaUnitTest with Inside {
       .runWith(Sink.queue())
 
     f.leaderStateEventsInput.offer(LeadershipTransition.Standby)
+    inside(events.pull().futureValue) {
+      case Some(Some(tpe)) => tpe shouldBe "event_stream_attached"
```

Comment:
The tests fails here. This was added with https://github.com/mesosphere/marathon/commit/fc863df666242fcd95d69c2d0af62d59ff2db092. @timcharper, is this still relevant?